sql/pre_install/tables.sql

```diff
@@ -466,24 +473,6 @@ CREATE TABLE _timescaledb_catalog.compression_algorithm (
   CONSTRAINT compression_algorithm_pkey PRIMARY KEY (id)
 );
 
-CREATE TABLE _timescaledb_catalog.hypertable_compression (
-  hypertable_id integer NOT NULL,
-  attname name NOT NULL,
-  compression_algorithm_id smallint,
-  segmentby_column_index smallint,
-  orderby_column_index smallint,
-  orderby_asc boolean,
-  orderby_nullsfirst boolean,
-  -- table constraints
-  CONSTRAINT hypertable_compression_pkey PRIMARY KEY (hypertable_id, attname),
-  CONSTRAINT hypertable_compression_hypertable_id_orderby_column_index_key UNIQUE (hypertable_id, orderby_column_index),
-  CONSTRAINT hypertable_compression_hypertable_id_segmentby_column_index_key UNIQUE (hypertable_id, segmentby_column_index),
-  CONSTRAINT hypertable_compression_compression_algorithm_id_fkey FOREIGN KEY (compression_algorithm_id) REFERENCES _timescaledb_catalog.compression_algorithm (id),
-  CONSTRAINT hypertable_compression_hypertable_id_fkey FOREIGN KEY (hypertable_id) REFERENCES _timescaledb_catalog.hypertable (id) ON DELETE CASCADE
-);
```

Comment:
Moving this data to the `hypertable` table seems like a very complicated change compared to just replacing the `hypertable_id` with, for example, a `regclass` or a `name` and (optionally) rename the table.

Reply:
Yes, I realize that most of the changes are from the move of the columns, but this does not seem warranted since you can accomplish the same with a smaller change without limiting compression to not support anything but hypertables.

Also, moving data around like this runs the risk of creating problems when upgrading.

Reply:
I changed it to store compression settings in dedicated table.